iroha_macro/iroha_derive/tests/log.rs

```diff
@@ -0,0 +1,27 @@
+#[cfg(test)]
+mod tests {
+
+    use iroha_derive::*;
+
+    #[log]
```

Comment:
Seems like there is no straightforward way to test this macro as we need to check stdout

Reply:
Maybe use https://crates.io/crates/gag . But it has a limitation that it runs only on unix systems.

Reply:
I think it's okay for now to only compile this and check in other tests